### PR TITLE
Set-up Matomo analytics

### PR DESCRIPTION
**Pull request by lebaudantoine**
## Purpose

Offer analytics on page's visits to the communication/marketing team.

## Proposal

- [X] Setup Matomo per the React integration guide.
- [X] Add the official Iframe to allow user Opt Out tracking analytics.
- [X] Update Data and cookies page's content.
- [X] Minor changes in Legal pages UI.

### CODE REVIEW

File: src/pages/suivi/index.tsx [hunk 1]
```diff
@@ -27,16 +46,51 @@ export default function Suivi() {
...
+              libre
+            </ExternalLink>
+            , paramétré pour être en conformité avec la{" "}
+            <ExternalLink
+              href="https://www.cnil.fr/fr/cookies-et-autres-traceurs/regles/cookies-solutions-pour-les-outils-de-mesure-daudience"
+              className="underline decoration-1 hover:decoration-2 transition ease-in-out delay-50 duration-300 underline-offset-4"
+              title="recommandation « Cookies »"
+            >
+              recommandation « Cookies »
+            </ExternalLink>{" "}
```
Comment (lebaudantoine) [inline src/pages/suivi/index.tsx:72]: @manuhabitela please note this `ExternalLink` will need to be refactored when rebasing my work with your recent works on title. That's why I didn't spend much time on it.

Also I would like to introduce a wrapper, to have a unique component for external link in legal pages.

Comment (lebaudantoine) [reply src/pages/suivi/index.tsx:72]: Might be, let's see after your PR

Comment (manuhabitela) [reply src/pages/suivi/index.tsx:72]: You can remove the `title` props that are not necessary anymore.

File: src/pages/suivi/index.tsx [hunk 2]
```diff
@@ -27,16 +46,51 @@ export default function Suivi() {
...
           </SubTitle>
           <Text>
-            Les statistisques d&apos;usage de nos produits ne sont pas encore en
-            accès libre.
+            Bien sûr ! Les statistiques d’usage de la majorité de nos produits,
+            dont {URL_SITE}, sont disponibles en accès libre sur{" "}
+            <ExternalLink
+              href="https://stats.data.gouv.fr/index.php?module=CoreHome&action=index&idSite=21&period=range&date=previous30#?period=range&date=previous30&module=VisitsSummary&action=index&idSite=21&category=Dashboard_Dashboard&subcategory=1"
+              className="underline decoration-1 hover:decoration-2 transition ease-in-out delay-50 duration-300 underline-offset-4"
+              title="stats.data.gouv.fr"
```
Comment (manuhabitela) [inline src/pages/suivi/index.tsx:89]: same here, no need anymore!

**Review by manuhabitela — APPROVED** [review]

**Comment by manuhabitela** [comment]
i'm currently changing the default styles with DSFR so don't worry about styling things of external links I'll fix it :)

**Pull request merged**